Duplicati.Library.RestAPI/Strings.cs

```diff
@@ -29,8 +29,9 @@ public static class Program
         public static string WrongSQLiteVersion(System.Version actualversion, string expectedversion) { return LC.L(@"Unsupported version of SQLite detected ({0}), must be {1} or higher", actualversion, expectedversion); }
         public static string WebserverWebrootDescription { get { return LC.L(@"The path to the folder where the static files for the webserver is present. The folder must be located beneath the installation folder."); } }
         public static string WebserverPortDescription { get { return LC.L(@"The port the webserver listens on. Multiple values may be supplied with a comma in between."); } }
-        public static string WebserverCertificateFileDescription { get { return LC.L(@"The certificate and key file in PKCS #12 format the webserver use for SSL. Only RSA/DSA keys are supported."); } }
-        public static string WebserverCertificatePasswordDescription { get { return LC.L(@"The password for decryption of certificate PKCS #12 file."); } }
+        public static string WebserverUseHTTPSDescription { get { return LC.L(@"If provided, determine whether to use HTTPS. If not provided, compatibility mode determine based on certificate presence about usage of HTTPS."); } }
+        public static string WebserverCertificateFileDescription { get { return LC.L(@"The certificate and key file in PKCS #12 format the webserver use for SSL."); } }
+        public static string WebserverCertificatePasswordDescription { get { return LC.L(@"The password for decryption of certificate PKCS #12 file. USE STRONG PASSWORD, CERTIFICATE FILE WILL BE SAVED TO DRIVE ENCRYPTED BY CERTIFICATE PASSWORD!"); } }
```

Comment:
Could we use a random generated passphrase for encrypting the certificate?
If the user has placed a file, they can use as sloppy a password as they like.
If the user uploads the certificate, we only need the password for decrypting, but we could save it with a strong randomly generated password.

Also, why do we need to store it as a file? Would it be sufficient to store it in the database?

Reply:
user will provide certificate file in pfx form, which has to have password by definition (could be empty). I like proposed idea (was thinking in same way) but that would need to reencrypt private key with generated password. I didnt find way how to import, reecrypt, export certificate with EC keys without new libraries. :/ I will look again, there was some bug in past with openssl(linux version).

Reply:
Storing certificate to file is only was to pass certificate to kestrel, because kestrel fail to load certificate with EC key from X509Certificate2 instance.

Reply:
found a way to reencrypt, that would simplify code

Reply:
will be fixed in next commit, guid is generated as password for storing in DB

Reply:
fixed

Reply:
password is unique generated guid